composer.json

```diff
@@ -20,7 +20,8 @@
 		"composer/installers": "~1.0",
 		"monolog/monolog": "~1.11",
 		"league/flysystem": "~1.0.12",
-		"symfony/yaml": "~2.7"
+		"symfony/yaml": "~2.7",
+		"guzzlehttp/guzzle": "~5.3|~6.0.1|~6.1"
```

Comment:
What’s the thinking behind these constraints? Given that there could be breaking changes between versions 5 and 6 (I haven’t checked, but semver yo), shouldn’t we pick a specific major version?